src/components/input/index.js

```diff
@@ -71,9 +71,7 @@ const Input = ({
             name={name}
             disabled={disabled}
           >
-            <svg className={styles['show-password-icon']} viewBox={toggleIcon.viewBox}>
-              <use xlinkHref={`#${toggleIcon.id}`} />
-            </svg>
+            <img src={toggleIcon} className={styles['show-password-icon']} alt={`${id} icon`} />
```

Comment:
alt text should be relevant text that describes the image. The alt text is what's read by screen readers and is what will be displayed in case the image doesn't load.
I don't think "id" would be a good alt